java-client/src/main/java/org/opensearch/client/opensearch/core/search/Hit.java

```diff
@@ -265,8 +267,19 @@ public final Long version() {
 
     /**
      * API name: {@code sort}
+     *
+     * <p><b>NOTE: In version 3.0.0 of opensearch-java, this method will instead return a {@code List<FieldValue>}.</b></p>
      */
     public final List<String> sort() {
+        return this.sort.stream().map(FieldValue::_toJsonString).collect(Collectors.toList());
+    }
+
+    /**
+     * API name: {@code sort}
+     *
+     * <p><b>NOTE: In version 3.0.0 of opensearch-java, this method will be renamed to replace {@link #sort()}.</b></p>
+     */
+    public final List<FieldValue> sortVals() {
```

Comment:
```suggestion
    public final List<FieldValue> sortFields() {
```

Reply:
For 3.x (as you mentioned) we could replace `sort()` directly

Reply:
@reta I can change this naming, however it then won't match that already used to do this fix for `SearchRequest.searchAfter`: 
https://github.com/opensearch-project/opensearch-java/blob/2.x/java-client/src/main/java/org/opensearch/client/opensearch/core/SearchRequest.java#L741-L743

Reply:
@Xtansia ah, I see, sure, let's be consistent, thanks @Xtansia